script/proof

```diff
@@ -32,4 +32,4 @@ bundle exec jekyll build -s $SOURCE -d $DESTINATION --trace
 
 # 3.
 msg "Proofing..."
-time bundle exec htmlproofer ./$DESTINATION --url-ignore $INGORE_HREFS $@
+time bundle exec htmlproofer ./$DESTINATION --allow-hash-href --disable-external --url-ignore $IGNORE_HREFS $@
```

Comment:
I'm not sure about `--disable-external` – I'd like to know when we're pointing to resources that are no longer there. What is failing? Can we periodically nix them?